sailthru_content/sailthru_content.py

```diff
@@ -0,0 +1,317 @@
+#! /usr/bin/env python
+
+""" Synchronize edX course catalog in to Sailthru. """
+
+import logging
+import datetime
+import json
+import sys
+import os
+import argparse
+from edx_rest_api_client.client import EdxRestApiClient
+from sailthru.sailthru_client import SailthruClient
+
+logger = logging.getLogger(__name__)
+
+
+def setup_logging():
+    """ Setup logging.
+
+    Add support for console logging, and avoid truncation by pandas.
+    """
+    msg_format = '%(asctime)s - %(levelname)s - %(message)s'
+    ch = logging.StreamHandler()
+    ch.setFormatter(logging.Formatter(msg_format))
+    ch.setLevel(logging.DEBUG)
+    logger.addHandler(ch)
+    logger.setLevel(logging.DEBUG)
```

Comment:
As the writer of those other scripts, I don't recommend trying to be consistent with them. They could probably use some TLC.